bump.lua

```diff
@@ -467,7 +501,7 @@ function World:project(item, x,y,w,h, goalX, goalY, filter)
   local dictItemsInCellRect = getDictItemsInCellRect(self, cl,ct,cw,ch)
 
   for other,_ in pairs(dictItemsInCellRect) do
```

Comment:
Again, see World:check() / World:projectMove() for more details on how alreadyVisited works.